src/07-even-more-loops.js

```diff
@@ -44,7 +62,15 @@ function shoutWhileLoop(array) {}
  *  shoutForOfLoop([ "A", "Very", "Happy", "Array" ]);
  *  //> [ "A!", "Very!", "Happy!", "Array!" ];
  */
-function shoutForOfLoop(array) {}
+function shoutForOfLoop(array) {
+  let i = 0;
```

Comment:
the i is totally not necessary here. remember this for...of loop will do this entire iteration for us. Refactor this so that we no longer use i and instead simply use the element.